openhcl/virt_mshv_vtl/src/processor/mod.rs

```diff
@@ -266,6 +262,13 @@ pub trait HardwareIsolatedBacking: Backing {
     fn cvm_state_mut(&mut self) -> &mut crate::UhCvmVpState;
     /// Gets CVM specific partition state.
     fn cvm_partition_state(&self) -> &crate::UhCvmPartitionState;
+    /// Copies shared registers (per VSM TLFS spec) from the source VTL to
```

Comment:
We can always move it back if need be. But we probably should invest a little more time in seeing if we can get some way to turn a Backing into an Option<HardwareIsolatedBacking> so that all of these sorts of cases can be only here and we can avoid stubs.

Reply:
Not having any success here...